src/site/content/en/angular/angular.11tydata.js

```diff
@@ -22,6 +22,7 @@ module.exports = {
       {
         title: 'Fast',
         pathItems: [
+          'route-level-code-splitting',
```

Comment:
@mgechev should this post go after the one on performance budgets.

I noticed at the end of this post there is a warning which mentions:

```
Warning: Code-splitting can significantly improve an app's initial load time, but it comes at the cost of slowing down subsequent navigation. In the next post on route preloading you'll see how to work around this problem!
```

But with this setup the next post will be performance budgets, not preloading.